.github/workflows/unittests.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     strategy:
       matrix:
         python-version: [ '3.9', '3.10' , '3.11']
+        pandas-versions: [ '==1.5.2', '>=2.2.0']
```

Comment:
```suggestion
        pandas-versions: [ 'pandas==1.5.2', 'pandas>=2.2.0']
```
Then it will be more clear in the UI to see pandas is the package being constrained there. 
<img width="315" alt="Screenshot 2024-05-07 at 15 01 24" src="https://github.com/volueinsight/volue-insight-timeseries/assets/8268223/98b4f32d-906c-4bf3-81dd-45ebea709256">